_posts/2020-01-16-cycle-8-results.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Bisq DAO Cycle 8: Results"
+author: Steve Jain
+excerpt: "Cycle 8 of the Bisq DAO ended at block 609 186 on December 21 2019. This post covers its results. <br><br>"
+en-only: true
+---
+
+This post summarizes the results of Cycle 8 of the Bisq DAO.
+
+It comes a bit later than usual, as Cycle 8 wasn't a pretty one. During the cycle, some significant issues were made apparent in the human side of the network: a lack of management and budgeting resulted in suboptimal resource allotment needed for growth, which resulted in a ballooning BSQ supply, which resulted in an erosion of confidence in BSQ, which ultimately resulted in a sharp downturn in the BSQ price.
```

Comment:
> which resulted in an erosion of confidence in BSQ, which ultimately resulted in a sharp downturn in the BSQ price.

Not sure I'd state this so confidently. It may be part of the cause, and certainly would be part of the cause in the long run, but I don't think we have enough time or data to state this quite so definitively.

Reply:
I tweaked it in the new commit to "a lack of management and budgeting resulted in suboptimal resource allotment needed for growth, which seemed to reflect in a lower BSQ price."

Let me know if that's closer to what you had in mind.